server/api/app.go

```diff
@@ -69,6 +131,7 @@ var DefaultBindingsCall = &Call{
 
 type App struct {
 	Manifest *Manifest `json:"manifest"`
```

Comment:
4/5 We should "decouple" `Manifest` and `App`. Manifest is "listing data", static, same for all instances, immutable, comes from the marketplace via the env variable, and is only cached in memory. `App` is the local app's configuration, and is stored in the config. No need to store the manifest in the config, in fact we specifically want to avoid this (size limitations).

Reply:
Understood, should this be a the part of another ticket? Or should I include it in this PR?

Reply:
1/5 I'd say in this PR since in practical terms - can we even fit the mappings into the config? Isn't there a limit on the config value size?